tests/js-custom-authorization/custom_authorization.py

```diff
@@ -310,7 +312,7 @@ def create_keypair(local_id, valid_from, validity_days):
     with primary.client(local_user_id) as c:
         r = c.get("/app/cert")
         assert r.status_code == HTTPStatus.UNAUTHORIZED, r
-        assert "Not After" in r.body.json()["error"]["details"][0]["message"], r
+        assert "Not After" in parse_error_message(r), r
```

Comment:
I am not sure what this asserts, but it's probably not the right thing, because the exp message does not contain "Not After": "Current time {} is after token's Expiration Time (exp) claim {}"